Makefile

```diff
@@ -27,7 +27,7 @@ preview:
 	      _site/*/topics/categories/index.html \
 	      _site/*/topics/dates/index.html \
 	      _site/*/topics/index.html
-	bundle exec jekyll serve $(JEKYLL_FLAGS)
+	bundle exec jekyll serve --host 0.0.0.0 $(JEKYLL_FLAGS)
```

Comment:
What is the motivation for this addition?

Reply:
Whoops, ,I didn't mean to push that.  It solved a build issue for me on my travel laptop.  It would be easiest for me if we kept this, but if it's causing problems for anyone else, I'm happy to revert.

Reply:
No problems for me with using that host

Reply:
It worked fine for me as well when I built locally.